tests/test_utils.py

```diff
@@ -281,3 +281,224 @@ def dir_name_to_display_name(dir_name: str):
         )
         == "DISPLAY NAME TEST"
     )
+
+
+# Basic objects with only the attributes needed for `get_all_included_scenarios`
+# Can't use @dataclass since we run tests with older versions of python than 3.7
+class Feature:
+    """Bare minimum test feature class."""
+
+    def __init__(self, tags, scenarios):
+        self.tags = tags
+        self.scenarios = scenarios
+
+
+class Scenario:
+    """Bare minimum test scenario class."""
+
+    def __init__(self, tags, examples):
+        self.tags = tags
+        self.examples = examples
+
+
+class Example:
+    """Bare minimum test examples table class."""
+
+    def __init__(self, tags):
+        self.tags = tags
+
+
+arbitrary_multiple_number = 4
+tagless_scenarios = [
+    Scenario(tags=[], examples=[]) for _ in range(arbitrary_multiple_number)
+]
+feature_tagged_with_a = Feature(scenarios=tagless_scenarios, tags=["a"])
```

Comment:
:-) This threw me, thinking that there was something missing after `_a` instead of `a` being an arbitrary tag :-)

Reply:
Ha yeah maybe `x` and `y` would make more sense to avoid that perception. I definitely toyed with how to handle these so I wouldn't have to create a new series of nested features/scenarios/examples.